infra-templates/network-services/2/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '2'
+services:
+  network-manager:
+    image: rancher/network-manager:v0.2.14
+    privileged: true
+    network_mode: host
+    pid: host
+    command: plugin-manager --metadata-url http://rancher-metadata/2016-07-29
+    dns: 169.254.169.250
+    environment:
+      DOCKER_BRIDGE: docker0
+      METADATA_IP: 169.254.169.250
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock
+    - /var/lib/docker:/var/lib/docker
```

Comment:
Won't this line create same problems like we had previously on deployments with non standard docker root path(like `/data/docker`?
Maybe we could store somehow `ROOTBIND` passed to agents from 1.1.1-rc1 and expose as `{{ROOTBIND}}`